docker/Dockerfile.ngen

```diff
@@ -75,19 +57,19 @@ ARG REPO_URL \
     BUILD_PET="true" \
     BUILD_SLOTH="true"
 
-COPY --chown=${USER} --from=rocky_init_repo ${WORKDIR}/ngen ${WORKDIR}/ngen
-COPY --chown=${USER} --from=troute ${WORKDIR}/t-route/wheels /tmp/t-route-wheels
```

Comment:
Line 79 and 80 are to be recovered, because we need troute image with tag-name that is built with the same tag in this build.